core/ui.js

```diff
@@ -268,30 +274,38 @@ function generateDropContent(options)
                         </div> 
                     </div> 
                             
-                    <div id='incognito-option-presence-updates' class='incognito-options-item' style='cursor: pointer;'>
+                    <div id='incognito-option-online-status' class='incognito-options-item' style='cursor: pointer;'>
                         <div class='checkbox-container-incognito' style=''>
-                            <div class='checkbox checkbox checkbox-incognito ${presenceUpdatesChcekbox}
+                            <div class='checkbox checkbox checkbox-incognito ${onlineUpdatesCheckbox}
                             </div>
                         </div>
-                        Don't send \"Last Seen\" and \"Typing\" updates
-                        <div class='incognito-options-description'>${presenceCaption}</div>
+                        Hide \"Online\" status
```

Comment:
Let's have the titles as variables to make it consistent with the other titles

Reply:
Done! I added the read confirmations title to the variables list, they were missing too.